templates/logging-sidecar-configmap.yaml

```diff
@@ -33,10 +33,28 @@ data:
           workspace: "${WORKSPACE:--}"
           release: "${RELEASE:--}"
 
+
+      filter_common_logs:
+        type: filter
+        inputs:
+          - transform_syslog
+        condition:
+          type: "vrl"
+          source: '!includes(["worker"], .component)'
+
+      filter_task_logs:
+        type: filter
+        inputs:
+          - transform_syslog
+        condition:
+          type: "vrl"
+          source: 'includes(["worker"], .component)'
```

Comment:
Does this approach also work for the workers log? I'm thinking we might need to be a little more specific with our condition?

Reply:
Yes - i have tested this change

Reply:
transform_webserver:
        type: filter
        inputs:
          - log_files
        condition: '.component == "worker"'

Reply:
with vrl
```
transform_worker:
        type: filter
        inputs:
          - transform_syslog
        condition: 
          type: "vrl"
          source: '.component == "worker"'
```

Reply:
Had a chat with @pgvishnuram and both worker logs and task logs need to follow the 'task log' path here because they are both in json format.